public/apps/configuration/utils/form-row.tsx

```diff
@@ -37,7 +37,7 @@ export function FormRow(props: FormRowDeps) {
             <b>{props.headerText}</b>
             <i>{props.optional && ' - optional'}</i>
           </EuiText>
-          <EuiText color="subdued" size="xs">
+          <EuiText color="subdued" size="xs" className="form-row-subtext">
```

Comment:
This change will reduce width of small subtext as well. For example, see the Name and Anonymization subtext. width should be 75% of main panel width. This change is taking 75% width of Label component. Can you confirm with UX team if that's fine?